Client/Frontend/Browser/ResourceDownloadManager.swift

```diff
@@ -60,7 +60,7 @@ class ResourceDownloadManager: TabContentScript {
     static func downloadResource(for tab: Tab, url: URL) {        
         let token = UserScriptManager.securityToken.uuidString.replacingOccurrences(of: "-", with: "", options: .literal)
 
-        tab.webView?.evaluateJavaScript("D\(token).download(\"\(url)\");", completionHandler: { _, error in
+        tab.webView?.evaluateSafeJavascript(functionName: "D\(token).download", args: [url], sandboxed: false, completion: { _, error in
```

Comment:
Does this new `args` parameter accept non-JSON types such as this `URL` type? Seems previously it was reflected to a `String`. We may want to do `[url.absoluteString]` instead

Reply:
done! great catch here